extras/rya.console/src/main/java/mvm/rya/shell/command/mongo/administrative/MongoInstall.java

```diff
@@ -134,36 +133,26 @@ private RyaDetails initializeRyaDetails(final String instanceName, final Install
             throws AlreadyInitializedException, RyaDetailsRepositoryException {
         final RyaDetailsRepository detailsRepo = new MongoRyaInstanceDetailsRepository(getClient(), instanceName);
 
-        // Build the PCJ Index details.
+        // Build the PCJ Index details. [not supported in mongo]
```

Comment:
I think we should throw an exception if the user is trying to install an instance with PCJ turned on while it's not supported.